### ISSUE TITLE
WordArray appears to skip arrays with special characters

### ISSUE THREAD

**Issue by createdbypete** [rubocop/rubocop#1147]
I could be mis interpreting the rules but take this example from a [Jekyll test](https://github.com/jekyll/jekyll/blob/master/test/test_filters.rb#L185) I've been working on:

``` ruby
# Cyrillic
assert_equal ["ВУЗ", "Вуз", "вуз"], @filter.sort(["Вуз", "вуз", "ВУЗ"])
assert_equal ["_вуз", "вуз", "вуз_"], @filter.sort(["вуз_", "_вуз", "вуз"])
# Hebrew
assert_equal ["אלף", "בית"], @filter.sort(["בית", "אלף"])

# Another example using latin characters mixed with words
["fooê", "foö", "héllo"]
```

These lines do not get flagged as requiring `%w()` treatment.

```
$ rubocop -V
0.23.0 (using Parser 2.1.9, running on ruby 2.1.2 x86_64-darwin13.0)
```
